drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilitiesTest.java

```diff
@@ -125,6 +132,148 @@ public void testCanAcceptOtherwiseWrongOperator() throws Exception {
         assertFalse(ColumnUtilities.canAcceptOtherwiseValues(column));
     }
 
+    @Test
```

Comment:
Also applied. I am just wondering why I always forget to such obvious things. Anyway, thanks. Lets wait if jenkins will be green.